packages/app/src/crash-dump/actions/send-crash-dump-data.action.ts

```diff
@@ -36,7 +41,26 @@ export const sendCrashDumpData = createAsyncThunk(
       )
     }
 
-    const attachments = mapToAttachments(state.crashDump.data.downloadedFiles)
+    const files = await downloadingLogs()
+
+    const buffer = await archiveFiles({ files })
+
+    if (buffer === undefined) {
+      return rejectWithValue(
+        new SendingCrashDumpError(
+          "Create Crash Dump Ticket - ArchiveFiles error"
+        )
+      )
+    }
+
+    const logsAttachments = [
+      new File([buffer], attachedFileName, { type: "application/zip" }),
+    ]
+
+    const crashDumpAttachments = mapToAttachments(
+      state.crashDump.data.downloadedFiles
+    )
+    const attachments = logsAttachments.concat(crashDumpAttachments)
```

Comment:
🟢
I prefer spread instead of contact just for project consistency. However, the 
both solutions in this use case are good, so if don't see worth please skip my suggestion.

```suggestion
    const attachments = [...logsAttachments, ...crashDumpAttachments]
```

Reply:
Changed in [9f185d4](https://github.com/mudita/mudita-center/pull/586/commits/9f185d4bd7b2891cecf5832c9dbe6ecf43e73203)